backend/src/main/kotlin/org/globe42/web/persons/ChildDTO.kt

```diff
@@ -0,0 +1,17 @@
+package org.globe42.web.persons
+
+import org.globe42.domain.Child
+import org.globe42.domain.Location
+import java.time.LocalDate
+
+/**
+ * DTO for a [Child]
+ * @author JB Nizet
+ */
+data class ChildDTO(
+    val firstName: String?,
+    val birthDate: LocalDate?,
+    val location: Location
```

Comment:
The idea is to specify the children in France and abroad. I guess the location is always known, and always discloseable. We already have the number of children in both locations (since what mattered until now was the number of children in both locations), and we can thus migrate the current data to the new schema. But we don't have the first name nor the birth date yet, so I had to leave them as nullable (although I guess it won't be a problem to ask for those two information). We can refine that later if needed.